src/test/userTestSuccess.js

```diff
@@ -0,0 +1,62 @@
+// Testing modules
+import chai from "chai";
+import chaiHttp from "chai-http";
+import * as base from "../index.js";
+
+let should = chai.should();
+let userId;
+chai.use(chaiHttp);
+
+describe("user", () => {
+  // beforeEach(async () => {
+  //   try {
+
+  //     userId = null;
+  //   } catch (error) {}
+  // });
+
+  //Create a user
+
+  describe("POST Register", () => {
+    it("It should return a created user as an object", async () => {
+      let newUser = {
+        email: "info@gmail.com",
+        password: "Fab123",
+        lastName: "Fabrice",
+        firstName: "Hafashimana",
+      };
+      chai
+        .request(base.app)
+        .post(base.apiRoute + "/accounts/signup")
+        .send(newUser)
+        .end((err, res) => {
+          res.should.have.status(400);
+          res.body.should.be.a("object");
```

Comment:
You should check the data in the data structure not just if the body is an object.

Reply:
Thank you. I am rewriting my tests to check for properties and values